src/app/(frontend)/globals.css

```diff
@@ -3,6 +3,8 @@
 @tailwind components;
 @tailwind utilities;
 
+@import url('https://fonts.googleapis.com/css2?family=Creepster&display=swap');
```

Comment:
I think that using the URL to load the font will make this slower or the font will show up delayed on slower browsers. We can download the font in a future PR. I think this is fine for V1 - just wanted to call this out